card/lib/card/auth.rb

```diff
@@ -64,6 +76,9 @@ def set_current_from_session
         current_id
       end
 
+      def set_current_from_token token, user_id
```

Comment:
Unused method argument - `token`. If it's necessary, use `_` or `_token` as an argument name to indicate that it won't be used. You can also write as `set_current_from_token(*)` if you want the method to accept any arguments but don't care about them.<br>Unused method argument - `user_id`. If it's necessary, use `_` or `_user_id` as an argument name to indicate that it won't be used. You can also write as `set_current_from_token(*)` if you want the method to accept any arguments but don't care about them.